newIDE/app/src/EventsFunctionsExtensionEditor/index.js

```diff
@@ -1029,6 +1040,11 @@ export default class EventsFunctionsExtensionEditor extends React.Component<
                     eventsFunction={selectedEventsFunction}
                     eventsBasedBehavior={selectedEventsBasedBehavior}
                     eventsBasedObject={selectedEventsBasedObject}
+                    eventsFunctionsContainer={
+                      (selectedEventsBasedEntity &&
```

Comment:
That's where a common interface would make the code cleaner, right?

Reply:
Indeed